packages/forma-36-website/src/components/Navigation.js

```diff
@@ -17,123 +17,126 @@ const styles = {
     padding-top: ${tokens.spacingM};
     border-right: 1px solid ${tokens.colorElementMid};
   `,
+
   navList: css`
     display: flex;
     flex-direction: column;
     border-top: 1px solid ${tokens.colorElementMid};
-    padding: ${tokens.spacingM} ${tokens.spacingM};
+    padding: ${tokens.spacingM} 0;
     height: calc(100vh - ${heightOfHeader + heightOfDocSearch}px);
     overflow-y: auto;
+    color: ${tokens.colorTextMid};
   `,
 
   list: css`
     flex: 1 1 0;
     list-style: none;
     padding: 0;
-    margin-top: ${tokens.spacingXs};
+    margin-top: 0;
   `,
 
-  listItem: css`
-    position: relative;
-    padding-left: 0;
-    margin-bottom: ${tokens.spacingXs};
-
-    ul {
-      position: relative;
-    }
+  link: css`
+    display: flex;
+    justify-content: space-between;
+    padding: ${tokens.spacingXs} ${tokens.spacingM};
+    color: ${tokens.colorTextMid};
+    text-decoration: none;
+    transition: background-color ${tokens.transitionDurationDefault}
+      ${tokens.transitionEasingDefault};
 
-    li {
-      padding-left: ${tokens.spacingL};
+    &:hover {
+      background-color: ${tokens.colorElementLightest};
```

Comment:
I think a shade darker here would be better - still accessible too (contrast ratio is 11.03:1)

```suggestion
      background-color: ${tokens.colorElementLight};
```

Reply:
I agree with @m10l